### ISSUE TITLE
Use with CRS.Simple coordinate system

### ISSUE THREAD

**Issue by janzikmund** [danwild/leaflet-velocity#35]
I want to use Velocity with Leaflet's [Custom Coordinate Reference System](https://leafletjs.com/examples/crs-simple/crs-simple.html), but even though something shows up, the results look far away from what they should, see a [JS Fiddle](https://jsfiddle.net/janzikmund/5barc0Lo/) of how far I got.

Basically I made a 10x10 Velocity grid, and also use a 1-10 x 1-10 custom coordinates grid in Leaflet. The biggest issues I see are:
- speed of particles is inverse to zoom, so the more I zoom in, the slower they become. Imo their speed should stay constant or become faster
- for some reason they only move vertically, but the dataset is taken from sample [Wind GBR](https://danwild.github.io/leaflet-velocity/wind-gbr.json) where they also move horizontally so not sure why I don't see it?

Basically I am wondering - is Velocity able to work with Simple Custom Reference System at all through supplying matching dataset JSON and tweaking its settings, or does it require basic Earth Map only? Thanks a lot

**Comment by gee1k** [comment]
I also encountered the same problem. Is there a solution?

**Comment by zjfcool** [comment]
Me too

**Comment by danwild** [comment]
CRS is now inherited from leaflet map options
